instrumentation/events/lttng-module/net.h

```diff
@@ -8,7 +8,106 @@
 #include <linux/skbuff.h>
 #include <linux/netdevice.h>
 #include <linux/ip.h>
+#include <linux/ipv6.h>
+#include <linux/tcp.h>
 #include <linux/version.h>
+#include <lttng-endian.h>
+#include <net/sock.h>
+
+#ifndef ONCE_LTTNG_NET_H
+#define ONCE_LTTNG_NET_H
+
+static inline unsigned char __has_network_hdr(struct sk_buff *skb) {
```

Comment:
So for structs, the { goes after the same line, but for functions it goes on the line below. You C people are so inconsistent :p